Reclaim/Application/Commands/Review/UpdateReviewCommand.cs

```diff
@@ -1,7 +1,14 @@
-﻿using Reclaim.Domain.Entities.Write;
+﻿using System.ComponentModel.DataAnnotations;
+using Reclaim.Domain.Entities.Write;
 
 namespace Reclaim.Application.Commands.Review;
 
 public class UpdateReviewCommand : ICommand<ReviewWriteEntity>
 {
+    [MaxLength(24)]
+    public required string ReviewId { get; set; } = string.Empty;
```

Comment:
Is there a specific reason we are initializing the property despite the fact that it is required?

```suggestion
    public required string ReviewId { get; set; }
```